### PR TITLE
Add error code description for 'Information schema is changed'

### PR DESCRIPTION
**Pull request by Benjamin2037**
### First-time contributors' checklist 

- [X] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add related error code for development reference.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [X] master (the latest development version)
- [X] v7.0 (TiDB 7.0 versions)
- [X] v6.6 (TiDB 6.6 versions)
- [X] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: metadata-lock.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ summary: 介绍 TiDB 中元数据锁的概念、原理、实现和影响。
     | `BEGIN;`                   |                                           |
     |                            | `ALTER TABLE t MODIFY COLUMN a CHAR(10);` |
     | `SELECT * FROM t;` (报错 `Information schema is changed`) |             |
+> **注意：**
+上述报错 error code 是 '8028' 供开发者参考。
```
Comment (Oreoxmt) [inline metadata-lock.md:69]:
```suggestion
    | `SELECT * FROM t;` (报错 `Error 8028: Information schema is changed`) |             |
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Review by wjhuang2016 — APPROVED** [review]

**Pull request merged**